Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -259,7 +259,7 @@ public void onClick(View view) {
                 // Define custom styel for the share sheet list view
                 //.setStyleResourceID(R.style.Share_Sheet_Style);
 
-                branchUniversalObject.showShareSheet(MainActivity.this, linkProperties, shareSheetStyle, new Branch.BranchLinkShareListener() {
+                branchUniversalObject.showShareSheet(MainActivity.this, linkProperties, shareSheetStyle, new Branch.BranchLinkShareListener2() {
```

Comment:
Can we name this new interface as `IChannelProperties` and builder method as `setChannelProperties(IChannelProperties)`